remote/advancedcontainers/engine-options.md

```diff
@@ -0,0 +1,63 @@
+
+![Docker Path setting](images/platform-options/docker-path-setting.png)
+
+However, certain tricks like [Docker-from-Docker do not work](https://github.com/containers/libpod/issues/4056#issuecomment-535511841) due to limitations in Podman. This affects the **Remote-Containers: Try a Development Container Sample...** and **[Remote- Containers: Clone Repository in Container Volume...](/docs/remote/containers.md#quick-start-open-a-git-repository-or-github-pr-in-an-isolated-container-volume)** commands.
+
+Docker Compose is also not supported by Podman.
+
+## Docker Contexts
+
+[Docker Contexts](https://docs.docker.com/engine/context/working-with-contexts/) allow you to interact with different hosts - you can set up contexts for each host and switch between them.
+
+You create new contexts with `docker context create`. The current context can be changed using `docker context use <context>`.
+
+The [Docker extension](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-docker) comes with `docker.host` (same as `DOCKER_HOST` env variable) and `docker.context` (same as `DOCKER_CONTEXT` env variable) user settings that are also honored by the Remote-Containers extension.
+
+> **Note**: The above settings are only visible when the Docker extension is installed. Without the Docker extension, Remote-Containers will use the current context.
+
+### Colima
+
+[Colima](https://github.com/abiosoft/colima) provides container runtimes on macOS and Linux. It's recommended to use Colima v0.2.2 or later so that VS Code can properly see containers running through Colima.
+
+Update settings in the Docker extension to use Colima:
+* `"docker.host": "unix:///Users/<username>/.colima/docker.sock",`
+* `"docker.context": "colima"`
+
+## Remote hosts
+
+Connecting to a remote host can enable you to connect to a remote environment with Docker or another supported engine, rather than needing to install the engine locally.
+
+### SSH
+You may use the [Remote - SSH extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-ssh) with Remote - Containers. This enables you just to have Docker or an alternate supported engine on your remote machine - Docker is not needed locally.
+
+You may learn more about this and other remote connection options in the [Develop on a remote Docker host doc](https://code.visualstudio.com/remote/advancedcontainers/develop-remote-host#_connect-using-docker-contexts).
+
+### Windows Subsystem for Linux (WSL)
+On Windows, you can use Docker installed in WSL when using the [Remote - WSL extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-wsl).
+
+You can also use [Distrod](https://github.com/nullpo-head/wsl-distrod) to install a WSL distro with systemd.
```

Comment:
Maybe link to Docker's documentation on installing Docker on Linux and explain that the problem is that the dockerd daemon won't start automatically due to the lack of systemd or any other system daemon.

I've had success with distrod, maybe there are easier ways to get this working though, like `sudo dockerd &` in `~/.profile` or something. (@Chuxel you mentioned starting dockerd manually?)

Reply:
Expanded the WSL section to include a link to installing Docker on Linux and additional context over the dockerd limitation.

@Chuxel let me know if you have any other ways you'd recommend documenting here.

@chrmarti let me know if you have additional feedback on this section with the changes!